WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -259,6 +259,14 @@ public string BuildTransaction(PaymentInfo[] payments, OutPoint[] coins, int? fe
 		return smartTx.Transaction.ToHex();
 	}
 
+	[JsonRpcMethod("payincoinjoin")]
```

Comment:
> Could the RPC check if the entered amount is above the minimum, and below the max coinjoin amount

No, it is not possible because the minimum is specified by the rounds' parameters and you don't know in what rounds your payments will be done.

> but I think we shouldn't do that as user can receive money in the meantime

Exactly